Userland/DevTools/UserspaceEmulator/SoftFPU.cpp

```diff
@@ -1319,12 +1319,13 @@ void SoftFPU::PSRLQ_mm1_mm2m64(const X86::Instruction&) { TODO_INSN(); }
 void SoftFPU::PSRLQ_mm1_imm8(const X86::Instruction&) { TODO_INSN(); }
 
 // DATA TRANSFER
-// void MOVD // (reg 2 reg)
-// void MOVD // (ld from mem)
-// void MOVD // (st to mem)
+void SoftFPU::MOVD_mm1_rm32(const X86::Instruction&) { TODO_INSN(); };
+void SoftFPU::MOVD_rm32_mm2(const X86::Instruction&) { TODO_INSN(); };
 
 void SoftFPU::MOVQ_mm1_mm2m64(const X86::Instruction&) { TODO_INSN(); }
-void SoftFPU::MOVQ_mm1_m64_mm2(const X86::Instruction&) { TODO_INSN(); }
+void SoftFPU::MOVQ_mm1m64_mm2(const X86::Instruction&) { TODO_INSN(); }
+void SoftFPU::MOVQ_mm1_rm64(const X86::Instruction&) { TODO_INSN(); }; // long mode
+void SoftFPU::MOVQ_rm64_mm2(const X86::Instruction&) { TODO_INSN(); }; // long mode
```

Comment:
This patch should also be a part of the initial FPU patch, since it doesn't do anything beyond stubbing out the MMX instructions.

Reply:
I think I should separate changes to different parts of the code as separated as possible, and this is a change Mainly for LibX86 and does not fully concern the Emulator